docs/docs/unit-testing.md

```diff
@@ -40,7 +40,7 @@ module.exports = {
     ".+\\.(css|styl|less|sass|scss)$": `identity-obj-proxy`,
     ".+\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": `<rootDir>/__mocks__/file-mock.js`,
   },
-  testPathIgnorePatterns: [`node_modules`, `.cache`, `public`],
+  testPathIgnorePatterns: [`<rootDir>/node_modules`, `<rootDir>/\\.cache`, `<rootDir>/public`],
```

Comment:
Why did you also change `node_modules` and `.cache`?

Generally speaking I don't think this is the best solution as users with monorepos will have their `.cache` and `public` folder not at the root but in nested directories. Reading [Jest's doc](https://jestjs.io/docs/en/configuration#testpathignorepatterns-arraystring) I'm not sure if including the rootDir part still makes it a regex

Reply:
Good point, `node_modules` doesn't need the `<rootDir>` prefix. neither does `.cache` since it has a specific dot prefix (which was wrong BTW, because it was being interpreted as "any character"). How about doing this:

```
[`node_modules`, `\\.cache`, `<rootDir>.*/public`]
```

Reply:
> which was wrong BTW, because it was being interpreted as "any character"

Thank you for catching that, too!